build.gradle

```diff
@@ -20,6 +20,7 @@ dependencies {
     testCompile 'org.junit.jupiter:junit-jupiter-api:5.6.0'
     implementation 'org.springframework.boot:spring-boot-dependencies:2.2.5.RELEASE'
     implementation 'org.springframework.boot:spring-boot-starter-web'
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.4.2'
```

Comment:
Please explain which of the version should be used and why.

Reply:
I added the Jupiter-api:5.4.2 in order to run the unit tests